source/MultiGrid/MultiGrid.test.js

```diff
@@ -88,6 +113,19 @@ describe('MultiGrid', () => {
       })))
       expect(rendered.querySelectorAll('.ReactVirtualized__Grid').length).toEqual(2)
     })
+
+    it('should render 4 Grids when configured for fixed columns and rows', () => {
+      let multiGrid
+      findDOMNode(render(getMarkup({
+        fixedColumnCount: 1,
+        fixedRowCount: 1,
+        ref: (ref) => {
+          multiGrid = ref
+        }
+      })))
+
+      multiGrid.recomputeGridSize()
```

Comment:
Hm. I see.

`expect(...).not.toThrowError(...)` is kind of the default behavior of any testing framework though so probably not worth explicitly adding.

Perhaps I was just confused by the name of this test. What makes it different than [this one](https://github.com/codingbull/react-virtualized/blob/9de4f2b786a61cfa3ad17dc2eb86358440aa6b65/source/MultiGrid/MultiGrid.test.js#L42-L62) for instance. You could update it to say something about `recomputeGridSize` specifically, since that's what it's testing, but I wonder if we couldn't do 1 better than that and assert that it _actually_ recomputes the inner `Grid` sizes (and resets `_leftGridWidth` and `_topGridHeight` like we chatted about above)?